src/server/lib/renderer.tsx

```diff
@@ -49,6 +48,8 @@ const clientStateFromRequestStateLocals = (
     clientHosts,
   };
 
+  pageData.browserName = browserName;
```

Comment:
could `browserName` be included in the `pageData` object before it even gets to this point, e.g. in a higher middleware, to stop the object mutation?

Reply:
you mentioned setting it only where it is required here: https://github.com/guardian/gateway/pull/53#discussion_r506412936

I don't mind which approach we use.

Reply:
https://github.com/guardian/gateway/pull/139/commits/6c6b3b095f5ae5d6fbc819594fe6308b5646d08a

Reply:
👍 in either case, some refactoring will probably be required either before/after @paddyo merges in #141